app/HutchAgent/Services/StatusReportingService.cs

```
@@ -8,19 +8,38 @@ namespace HutchAgent.Services;
 public class StatusReportingService
 {
   private readonly ILogger<StatusReportingService> _logger;
+  private readonly ControllerApiService _controllerApi;
 
-  public StatusReportingService(ILogger<StatusReportingService> logger)
+  public StatusReportingService(ILogger<StatusReportingService> logger, ControllerApiService controllerApi)
   {
     _logger = logger;
+    _controllerApi = controllerApi;
   }
 
-  public Task ReportStatus(string jobId, JobStatus type, string? message = null)
+  /// <summary>
+  /// Report status to the submission layer.
+  /// </summary>
+  /// <param name="jobId"></param>
+  /// <param name="type"></param>
+  /// <param name="message"></param>
+  public async Task ReportStatus(string jobId, JobStatus type, string? message = null)
   {
     // This is the most dummy version of this there could be <3
     _logger.LogInformation(
       "Job [{Id}] Status Report [{Type}]: {Message}",
       jobId, type.ToString(), message);
 
-    return Task.CompletedTask;
+    try
+    {
+      await _controllerApi.UpdateStatusForTre(jobId, type, message);
+    }
+    catch (InvalidOperationException e)
+    {
+      _logger.LogError(exception: e, message: e.Message);
+    }
+    catch (Exception e)
```

why are we silencing exceptions all the time? why not let them throw; they'll be caught and logged eventually.